src/tests/all-tests.js

```diff
@@ -0,0 +1,272 @@
+                            "message": "Hello, yourself"
+                        }
+                    }
+                }
+            }
+        },
+        "wildcard": {
+            "type": "gpii.express.tests.router.hello",
+            "options": {
+                "path":    "/wildcard/*",
+                "message": "Hello, wild world."
+            }
+        },
+        "static": {
+            "type": "gpii.express.router.static",
+            "options": {
+                path:    "/",
+                content: contentDir
+            }
+        },
+        "reqview": {
+            "type": "gpii.express.tests.router.reqview"
+        }
+    }
+});
+
+express.start(function(){
+    jqUnit.module("Testing express module stack...");
+
+    jqUnit.asyncTest("Testing the 'static' router module (index content)...", function() {
+        var options = {
+            url: express.options.config.express.baseUrl
+        }
+        request.get(options, function(error, response, body) {
+            jqUnit.start();
+
+            expressTests.isSaneResponse(jqUnit, error, response, body);
+
+            var indexRegexp = /body of the index/;
+            jqUnit.assertNotNull("The body should match the index content...", body.match(indexRegexp));
+        });
+    });
+
+    jqUnit.asyncTest("Testing the 'static' router module (custom content)...", function() {
+        var options = {
+            url: express.options.config.express.baseUrl + "custom.html"
+        };
+        request.get(options, function(error, response, body) {
+            jqUnit.start();
+
+            expressTests.isSaneResponse(jqUnit, error, response, body);
+
+            var customContentRegexp = /custom page/;
+            jqUnit.assertNotNull("The body should match the custom content...", body.match(customContentRegexp));
+        });
+    });
+
+    jqUnit.asyncTest("Testing the 'hello' router module...", function() {
+        var options = {
+            url: express.options.config.express.baseUrl + "hello"
+        };
+        request.get(options, function(error, response, body) {
+            jqUnit.start();
+
+            expressTests.isSaneResponse(jqUnit, error, response, body);
+
+            jqUnit.assertEquals("The body should match the configured content...", express.hello.options.message, body);
+        });
+    });
+
+    jqUnit.asyncTest("Testing a nested 'hello/world' router module...", function() {
+        var options = {
+            url: express.options.config.express.baseUrl + "hello/world"
+        };
+        request.get(options, function(error, response, body) {
+            jqUnit.start();
+
+            expressTests.isSaneResponse(jqUnit, error, response, body);
+
+            jqUnit.assertEquals("The nested body should match the configured content...", express.hello.world.options.message, body);
+        });
+    });
+
+    jqUnit.asyncTest("Testing a wildcard path (root)...", function() {
```

Comment:
These tests all follow a repetitive pattern of firing a request and verifying a response. Please refactor them into a declarative form that captures just the variable material - as for example seen in our transformation tests at https://github.com/fluid-project/infusion/blob/master/tests/framework-tests/core/js/ModelTransformationTests.js#L87